### PR TITLE
SplitButton: Add prop to make it possible to apply rootExposed to the full button even it it is split

### PR DESCRIPTION
**Pull request by montselozanod**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Add splitFullActive state to make whole splitButton look active if true

### CODE REVIEW

**Comment by montselozanod** [comment]
@micahgodbolt @dzearing @christiango  I added getClassNames func is a prop. Let me know what you guys think?

File: packages/office-ui-fabric-react/src/components/Button/examples/Button.Split.Example.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export class ButtonSplitExample extends React.Component<IButtonProps, {}> {
             text='Create account'
             onClick={ alertClicked }
             split={ true }
+            splitFullActiveState={ true }
```
Comment (christiango) [inline packages/office-ui-fabric-react/src/components/Button/examples/Button.Split.Example.tsx]: Let's remove this prop now that we have getClassNames

File: packages/office-ui-fabric-react/src/components/Button/BaseButton.classNames.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const getClassNames = memoizeFunction((
   disabled: boolean,
   checked: boolean,
   expanded: boolean,
-  isSplit: boolean | undefined
+  isSplit: boolean | undefined,
```
Comment (christiango) [inline packages/office-ui-fabric-react/src/components/Button/BaseButton.classNames.ts]: Don't need this comma right?

File: packages/office-ui-fabric-react/src/components/Button/Button.types.ts [hunk 3]
```diff
@@ -185,6 +186,22 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
...
  */
   data?: any;
+
+  /**
+  * Method to provide the classnames to style a button.
+  * The default value for this prop is the getClassnames func
+  * defined in BaseButton.classnames.
+  * @default getBaseButtonClassNames
+  */
+  getClassNames?: (styles: IButtonStyles,
```
Comment (christiango) [inline packages/office-ui-fabric-react/src/components/Button/Button.types.ts]: I wonder if we should not pass in the styles for getClassNames (like we do for contextual menu). If you are implementing getClassNames, you'll likely want to provide the styles yourself

Comment (montselozanod) [reply packages/office-ui-fabric-react/src/components/Button/Button.types.ts]: Yes I agree... let me remove them

**Pull request merged**